packages/serverless/package.json

```diff
@@ -41,7 +41,7 @@
     "typescript": "3.7.5"
   },
   "scripts": {
-    "build": "run-p build:cjs build:esm build:awslambda-layer",
+    "build": "run-p build:cjs build:esm && yarn build:awslambda-layer",
```

Comment:
How does `&&` interact with `run-p`?
a) `yarn build:awslambda-layer` runs after `run-p` has finished.
b) `yarn build:awslambda-layer` runs after `build:esm` has finished, potentially when `build:cjs` has not finished yet.

Didn't find the answer in the [docs](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-p.md).

Reply:
Building the aws lambda layer will occur after `run-p` has executed